framework/include/userobject/NodalUserObject.h

```diff
@@ -81,6 +69,9 @@ class NodalUserObject :
 
   /// Reference to current node pointer
   const Node * & _current_node;
+
+  // Flag for enable/disabling multiple execute calls on nodes that share block ids
+  const bool & _unique_block_execute;
```

Comment:
I'm not sure if I'm crazy about this name. "unique block" seems to suggest something with the blocks, not the nodes within the block. Maybe "unique_node_execute"?